mcrypt/mcrypt.php

```diff
@@ -363,7 +363,7 @@ function mcrypt_generic_deinit ($td) {}
  * @param resource $td <p>
  * The encryption descriptor.
  * </p>
- * @return int|bool If the self test succeeds it returns false. In case of an error, it
+ * @return bool If the self test succeeds it returns false. In case of an error, it
```

Comment:
According to the documentation and source code (https://www.php.net/manual/en/function.mcrypt-enc-self-test.php https://github.com/php/pecl-encryption-mcrypt/blob/master/mcrypt.c#L743) this function returns `int`